pkg/securitycontext/provider.go

```diff
@@ -95,3 +96,53 @@ func makeCapabilites(capAdd []api.Capability, capDrop []api.Capability) ([]strin
 	}
 	return addCaps, dropCaps
 }
+
+func determineEffectiveSecurityContext(pod *api.Pod, container *api.Container) *api.SecurityContext {
```

Comment:
If you have a non-nil but empty CSC, that means none of the values from the PSC apply. e.g. if you have PSC.RunAsUser=1234, you lose that because CSC.RunAsUser=nil wins.